### PR TITLE
jpeg2000 test

### PR DESCRIPTION
**Pull request by EricWVGG**
JPEG 2000 test, thank you for looking at it.

### CODE REVIEW

**Comment by modernizr-savage** [comment]
**Tests failed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **BUSTED**

Commit: 75ae1973c09dc2145ed11d92e032943f90455aec
Build details: https://travis-ci.org/modernizr-savage/Modernizr/builds/82039009

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

File: feature-detects/img/jpeg2000.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+/* DOC
+Test for JPEG 2000 support
+*/
+define(['Modernizr', 'addTest'], function(Modernizr, addTest) {
+
+  Modernizr.addAsyncTest(function() {
+    var image = new Image();
+
+    image.onload = image.onerror = function() {
+      addTest('jpeg2000', image.width == 1, {aliases: ['jpeg-2000']});
```
Comment (patrickkettner) [inline feature-detects/img/jpeg2000.js]: you do not need to add the alias here

**Comment by modernizr-savage** [comment]
**Tests failed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **BUSTED**

Commit: b7f986cfb196788a39a6c8e978bfc13e4a814f0e
Build details: https://travis-ci.org/modernizr-savage/Modernizr/builds/82436750

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

**Comment by EricWVGG** [comment]
wooooooo

**Comment by patrickkettner** [comment]
sorry to be a pest, but could you squash it down to a single commit? Then we will add 'er right in

**Comment by modernizr-savage** [comment]
**Tests failed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **BUSTED**

Commit: f18059aa35aa3f201688d92685c8df2920175129
Build details: https://travis-ci.org/modernizr-savage/Modernizr/builds/82564307

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

**Comment by patrickkettner** [comment]
You!!!

**Comment by EricWVGG** [comment]
^ savage is complaining, did I screw up?

**Comment by patrickkettner** [comment]
No, savage is messed up at the moment :(

**Comment by EricWVGG** [comment]
Got it.

Rad. Thank you so much for your patience. This was very exciting for me.


On September 28, 2015 at 11:05:48 AM, patrick kettner (notifications@github.com) wrote:

No, savage is messed up at the moment :(

**Comment by patrickkettner** [comment]
my absolute pleasure - I am stoked for you, too!

On Mon, Sep 28, 2015 at 8:06 AM, Eric Jacobsen <notifications@github.com>
wrote:

> Got it.
>
> Rad. Thank you so much for your patience. This was very exciting for me.
>
>
> On September 28, 2015 at 11:05:48 AM, patrick kettner (
> notifications@github.com) wrote:
>
> No, savage is messed up at the moment :(
>
> > On Sep 28, 2015, at 7:57 AM, Eric Jacobsen <notifications@github.com>
> wrote:
> >
> > ^ savage is complaining, did I screw up?
> >
> > —
> > Reply to this email directly or view it on GitHub.
> >
> —
> Reply to this email directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Modernizr/Modernizr/pull/1679#issuecomment-143769362>.
>



-- 
patrick

**Pull request merged**